tracer/src/Datadog.Trace/Util/SamplingHelpers.cs

```diff
@@ -16,5 +17,22 @@ internal static class SamplingHelpers
 
         internal static bool IsKeptBySamplingPriority(ArraySegment<Span> trace) =>
             trace.Array![trace.Offset].Context.TraceContext?.SamplingPriority > 0;
+
+        internal static int? GetDecisionMaker(TraceContext traceContext)
+        {
+            var decisionMaker = traceContext.Tags.GetTag(Tags.Propagated.DecisionMaker);
```

Comment:
Note that this tag is only added if the trace is sampled (sampling priority > 0). I don't know if or how this affects the rate and error samplers.